packages/ember/tailwind-webpack/files/config/postcss.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: {
     tailwindcss: {
-      config: 'app/config/tailwind.config.js',
```

Comment:
This is just because I needed to base my work on: https://github.com/NullVoxPopuli/ember-apply/pull/496
Once that PR is merged, this line will disappear.

Reply:
:tada: